bitnami/superset/Chart.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright Broadcom, Inc. All Rights Reserved.
+# SPDX-License-Identifier: APACHE-2.0
+
+annotations:
+  category: Analytics
+  licenses: Apache-2.0
```

Comment:
I'm missing the "image" annotation listing the images included in the chart.

Reply:
Isn't that added during the first automatic release?

Reply:
Given our "common.errors.insecureImages" helper relies on this information, I'd manually add it to ensure the mechanism works from the very beginning. Then, you can forget about this and rely on the automatic releases to update it.

Reply:
Unresolved topic